akka-stream-tests/src/test/scala/akka/stream/impl/StreamLayoutSpec.scala

```diff
@@ -124,21 +126,23 @@ class StreamLayoutSpec extends AkkaSpec {
     }
 
     "not fail materialization when building a large graph with simple computation" when {
+      // Seen tests run in 9-10 seconds, these test cases are heavy on the GC
```

Comment:
yes, I'm not suggesting to remove it, but we can keep it mind if we see spurious failures occurring right after this test

increase the PatienceConfig to 20.seconds